app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PostsController < ApplicationController
+  before_action :find_user, only: [:create]
```

Comment:
Don't need these `before_action`s if you only use them for one method, just go ahead and put them in the method